krausening-python/src/krausening/logging/log_manager.py

```diff
@@ -5,6 +5,16 @@ class LogManager:
     """
     Class for handling logging.
     """
+    DEFAULT_LOG_LEVEL = 'INFO'
+
+    LOG_LEVELS = {
+    'CRITICAL': logging.CRITICAL,
+    'ERROR': logging.ERROR,
+    'WARNING': logging.WARNING,
+    'INFO': logging.INFO,
+    'DEBUG': logging.DEBUG,
+    'NOTSET': logging.NOTSET,
+}
```

Comment:
C: These are the logging levels Python's `logging` library allows (https://docs.python.org/3/library/logging.html#logging-levels).